### PR TITLE
Created initial Sign In & Register pages, added more info to the header

### PR DESCRIPTION
**Pull request by lili2311**
[PR](https://womenhackfornonprofits.atlassian.net/browse/PIMP-70)
[Register](https://cloud.githubusercontent.com/assets/2911613/13031351/c6ba6d5c-d2c3-11e5-8049-0da5300e9d12.png)
[Sign in](https://cloud.githubusercontent.com/assets/2911613/13031353/c9571db2-d2c3-11e5-80b7-41ee04a44a82.png)

### CODE REVIEW

**Comment by lili2311** [comment]
@rimpey please review initial commit of both pages

File: frontend/src/pages/login.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    
+    <!--#include virtual="includes/head.html" -->
+    
+    <body>
+        <!--#include virtual="includes/header.html" -->
+        <div class="large-12 columns pnl-title pnl-summary">
+            <h1>Log In</h1>
+            <form id="sign-in" method="POST" action="" class="large-4 row">
+                <label for="email">Email:</label>
+                <input type="email" name="email" placeholder="Username" required>
```
Comment (tanyapowell) [inline frontend/src/pages/login.html:12]: Can you change the placeholder to `Email Address`? Just to make it explicitly clear it's the email address the user should enter

Comment (lili2311) [inline frontend/src/pages/login.html:12]: @tanyapowell I followed the wireframe exactly, I will check with Jeanette if that is OK as it is a visual change too.

Comment (tanyapowell) [inline frontend/src/pages/login.html:12]: That's cool! I had a conversation with my dad at lunchtime about filling in an online form and made me realise how important it is to be ridiculously clear on what you are asking for

**Comment by lili2311** [comment]
Opened issue for the [placeholder](https://github.com/womenhackfornonprofits/pimpmycause-rebuild/issues/10)

**Pull request merged**